xattr_darwin.go

```diff
@@ -2,72 +2,76 @@
 
 package xattr
 
-import "syscall"
+import (
+	"syscall"
+	"unsafe"
+)
 
-// Get retrieves extended attribute data associated with path.
-func Get(path, name string) ([]byte, error) {
-	// find size.
-	size, err := getxattr(path, name, nil, 0, 0, 0)
-	if err != nil {
-		return nil, &Error{"xattr.Get", path, name, err}
+func getxattr(path string, attr string, dest []byte) (sz int, err error) {
+	value, size := sliceToPtr(dest)
+	/*
+		ssize_t getxattr(const char *path, const char *name,
+			void *value, size_t size, u_int32_t position, int options)
+	*/
+	r0, _, e1 := syscall.Syscall6(syscall.SYS_GETXATTR, uintptr(unsafe.Pointer(syscall.StringBytePtr(path))),
+		uintptr(unsafe.Pointer(syscall.StringBytePtr(attr))), uintptr(unsafe.Pointer(value)), uintptr(size), 0, 0)
+	if e1 != 0 {
```

Comment:
lets keep verbose `e1 != syscall.Errno(0) `

Reply:
As you prefer, but note that stdlib does it like this
```go
	_, _, e1 := Syscall(SYS_REMOVEXATTR, uintptr(unsafe.Pointer(_p0)), uintptr(unsafe.Pointer(_p1)), 0)
	if e1 != 0 {
		err = errnoErr(e1)
	}
```
https://github.com/golang/go/blob/dc3a92edaabf012cabd90304bac1ed8fe3bb7444/src/syscall/zsyscall_linux_amd64.go#L771

Reply:
Hard coded `0` looks like a magic number. Moreover the file was autogenerated by:
`mksyscall.pl -tags linux,amd64 syscall_linux.go syscall_linux_amd64.go`